filter_linux.go

```diff
@@ -64,6 +64,9 @@ type Flower struct {
 	EncSrcIPMask  net.IPMask
 	EncDestPort   uint16
 	EncKeyId      uint32
+	IPProto       *nl.IPProto
+	DstPort       uint16
```

Comment:
For consistency can you name this one `DestPort` ? like `DestIP` and other Dest* attributes in this struct ?